fireworks_poe_bot/__main__.py

```diff
@@ -49,7 +54,16 @@ def main():
         else:
             assert k in ["print_supported_models"], f"Unknown argument {k}"
 
-    bot = FireworksPoeServerBot(args.model, args.environment, "0.0.1")
+    fireworks.client.api_key = server_args.fireworks_api_key
+    if server_args.fireworks_api_base_url:
+        fireworks.client.base_url = server_args.fireworks_api_base_url
```

Comment:
Let's just let these get picked up as env vars instead of passing them as commandline args

https://github.com/fw-ai/fireworks/blob/34d0c99ad914fd98c7e1aee959ac4d5903775f1f/public-py/fireworks/client/__init__.py#L9-L10